### PR TITLE
Python client library

### PR DESCRIPTION
**Pull request by kompotkot**
## Changes

Python client library and small fixes of API.

## How to test these changes?

Tested locally.

### CODE REVIEW

**Comment by kompotkot** [comment]
Resolves https://github.com/bugout-dev/entity/issues/7

File: clients/python/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# client
```
Comment (zomglings) [inline clients/python/README.md]: Suggest a new title: `Moonstream Entity Python Client`

File: clients/python/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Generate Moonstream access token at https://moonstream.to/account/tokens and add to environment variable:
```
Comment (zomglings) [inline clients/python/README.md]: In order to use this client, you will need a Moonstream access token. You can generate one at https://moonstream.to/account/tokens.

The client library reads this token from the `MOONSTREAM_ACCESS_TOKEN` environment variable. To set it:

```bash
export MOONSTREAM_ACCESS_TOKEN="<your access token>"
```

File: api/entityapi/actions.py [hunk 3]
```diff
@@ -38,7 +38,8 @@ def parse_entity_to_entry(
             try:
                 address = Web3.toChecksumAddress(cast(str, vals))
             except Exception:
-                logger.info(f"Unknown type of web3 address {address}")
+                logger.info(f"Unknown type of web3 address {vals}")
```
Comment (zomglings) [inline api/entityapi/actions.py:41]: logger.error?

**Review by zomglings — APPROVED** [review]
lgtm

Suggested some minor changes.

File: api/entityapi/actions.py (hunk 3 above)
Comment (kompotkot) [reply api/entityapi/actions.py:41]: Idea was to bypass this value and just notify us, that it is not ethereum like address, without errors

File: clients/python/README.md (hunk 2 above)
Comment (kompotkot) [reply clients/python/README.md]: Done

File: clients/python/README.md (hunk 1 above)
Comment (kompotkot) [reply clients/python/README.md]: Done

**Pull request merged**